datasets.py

```diff
@@ -97,17 +98,17 @@ def __getitem__(self, index):
 			transcript = transcript[0]
 			signal, sample_rate = audio.read_audio(audio_path, sample_rate = self.sample_rate, mono = self.mono, normalize = True, backend = self.audio_backend, duration=self.max_duration) if self.frontend is None or self.frontend.read_audio else (audio_path, self.sample_rate)
 
-			transcript = dict(dict(audio_name = os.path.basename(transcript['audio_path'])), **transcript)
+			transcript = dict(dict(audio_name = os.path.basename(transcript['audio_path'])), ref = transcript['ref'])
```

Comment:
Please use `transcripts.audio_name(...)` instead of `basename(...)`, let's modernize this code